### ISSUE TITLE
New npm release for react 15.0.1?

### ISSUE THREAD

**Issue by OAGr** [codesuki/react-d3-components#94]
Realize it was released recently, but on the other hand hopefully it wouldn't require a big change.  Thanks!

```
git:Graph-experiment npm install --save react-d3-components                         |
├── UNMET PEER DEPENDENCY react@15.0.1
├── react-d3-components@0.6.2
└── UNMET PEER DEPENDENCY react-dom@15.0.1

npm WARN react-d3-components@0.6.2 requires a peer of react@^0.14.0 but none was installed.
npm WARN react-d3-components@0.6.2 requires a peer of react-dom@^0.14.0 but none was installed.
```

**Comment by codesuki** [comment]
Thanks for the comment! I'll check if it works and make a new release.

**Comment by OAGr** [comment]
Just fyi, I'm using it for the scatterplot and t seemed to work fine for 15.0.1

**Comment by codesuki** [comment]
I released 0.6.3 and changed the required version to >= 0.14.0 which should fix your problem.
Would be great if you could confirm.

**Comment by crobinson42** [comment]
@OAGr can you confirm and close?
